### PR TITLE
ref#8: make english and vietnamese page

### PR DESCRIPTION
**Pull request by yosiyuki**
refs #8

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -105,13 +107,13 @@ <h1 class="font-semibold text-[20px] md:text-[28px]">TalentAsiaJobs</h1>
               <div class="hidden z-50 my-4 text-base list-none bg-white rounded divide-y divide-gray-100 shadow" id="dropdown">
                 <ul class="py-1 text-[14px] md:text-[16px]" aria-labelledby="dropdown">
                   <li>
-                    <a href="#" class="block py-2 px-4 text-gray-700 hover:bg-gray-100 dark:hover:bg-gray-600 dark:text-gray-200 dark:hover:text-white">日本</a>
+                    <a href="/" class="block py-2 px-4 text-gray-700 hover:bg-gray-100 dark:hover:bg-gray-600 dark:text-gray-200 dark:hover:text-white">日本</a>
```
Comment (yosiyuki) [inline index.html:110]: Please change 日本 -> 日本語

Comment (ttmthanh) [reply index.html:110]: I updated

Comment (ttmthanh) [reply index.html:110]: I updated

File: index.html [hunk 2]
```diff
@@ -105,13 +107,13 @@ <h1 class="font-semibold text-[20px] md:text-[28px]">TalentAsiaJobs</h1>
...
                   </li>
                   <li>
-                    <a href="#" class="block py-2 px-4 text-gray-700 hover:bg-gray-100 dark:hover:bg-gray-600 dark:text-gray-200 dark:hover:text-white">English</a>
+                    <a href="en" class="block py-2 px-4 text-gray-700 hover:bg-gray-100 dark:hover:bg-gray-600 dark:text-gray-200 dark:hover:text-white">English</a>
                   </li>
                   <li>
-                    <a href="#" class="block py-2 px-4 text-gray-700 hover:bg-gray-100 dark:hover:bg-gray-600 dark:text-gray-200 dark:hover:text-white">Tiếng Việt</a>
+                    <a href="vn" class="block py-2 px-4 text-gray-700 hover:bg-gray-100 dark:hover:bg-gray-600 dark:text-gray-200 dark:hover:text-white">Tiếng Việt</a>
```
Comment (yosiyuki) [inline index.html]: href="/vn" is better

Comment (ttmthanh) [reply index.html]: I updated

**Pull request merged**